src/ssis_migrator/parsers/config_parser.py

```
@@ -0,0 +1,390 @@
+#!/usr/bin/env python3
+"""
+Configuration Parser - Parse SSIS .dtsConfig files and handle environment variables
+"""
+
+import xml.etree.ElementTree as ET
+import os
+import base64
+from pathlib import Path
+from typing import Dict, List, Any, Optional
+from dataclasses import dataclass, field
+from cryptography.fernet import Fernet
```

Ruff (0.11.9)

`cryptography.fernet.Fernet` imported but unused

Remove unused import: `cryptography.fernet.Fernet`

(F401)